logical/framework/path_map.go

```diff
@@ -64,7 +64,7 @@ func (p *PathMap) pathStruct(s logical.Storage, k string) (*PathStruct, error) {
 		}
 	}
 	if salt != nil {
-		k = salt.SaltID(k)
+		k = salt.SaltIDHashFunc(k, "sha2-256", "-", saltpkg.SHA256Hash)
```

Comment:
Since this is hex formatted I think a simple `s` in front should suffice to differentiate.

Reply:
Done.